main.go

```diff
@@ -98,6 +101,9 @@ func setLangEnvVars(langStr string, cfg map[string]map[string]string) {
 	if requests, ok := cfg["requests"]; ok {
 		setLangEnvVarsForResource(langStr, "REQUEST", requests)
 	}
+	if runtime, ok := cfg["runtime"]; ok {
```

Comment:
nit: Can we rename this to `runtime_metrics`? Seeing `java.runtime.enabled` gives a different impression than `java.runtime_metrics.enabled`.